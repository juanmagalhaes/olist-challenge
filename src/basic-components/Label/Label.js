// @flow

import * as React from 'react';
import styles from './Label.module.scss';

type Props = {
  children: React.Node,
}

const Label = ({ children, ...rest }: Props) => (
  <label
    {...rest}
    className={styles.label}
  >
    {children}
  </label>
);

export { Label };

