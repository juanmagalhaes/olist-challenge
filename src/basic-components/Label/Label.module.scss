$label-color: #696D8C;
$label-line-height: 22px;

.label {
  color: $label-color;
  display: inline-block;
  line-height: $label-line-height;
}
