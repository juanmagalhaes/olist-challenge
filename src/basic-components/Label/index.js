// @flow

export { Label } from './Label';

