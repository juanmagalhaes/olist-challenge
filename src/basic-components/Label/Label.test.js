import React from 'react';
import { Label } from '~/basic-components/Label';

test('Label', () => expect(shallow(<Label>Full Name</Label>)).toMatchSnapshot())

