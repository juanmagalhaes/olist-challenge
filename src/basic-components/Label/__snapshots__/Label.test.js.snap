// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label 1`] = `
<label
  className="label"
>
  Full Name
</label>
`;
